### PR TITLE
Add sonar documentation

### PR DESCRIPTION
**Pull request by antarus**
close https://github.com/jhipster/jhipster-lite/issues/2543

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/generator/server/sonar/domain/SonarModulesFactory.java [hunk 1]
```diff
@@ -43,7 +43,9 @@ private JHipsterModuleBuilder commonModuleFiles(JHipsterModuleProperties propert
     return moduleBuilder(properties)
       .context()
       .put("sonarqubeDockerImage", dockerImages.get("sonarqube").fullName())
+      .put("srcMainDocker", "src/main/docker")
       .and()
+      .documentation(documentationTitle("sonar"), SOURCE.template("sonar.md"))
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/generator/server/sonar/domain/SonarModulesFactory.java:48]: you could add this file to unit test, to be sure it's generated

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

File: src/main/java/tech/jhipster/lite/generator/server/sonar/domain/SonarModulesFactory.java (hunk 1 above)
Comment (antarus) [reply src/main/java/tech/jhipster/lite/generator/server/sonar/domain/SonarModulesFactory.java:48]: yes

**Review by pascalgrimaud — APPROVED** [review]

**Comment by antarus** [comment]
@pascalgrimaud test add

**Comment by DamnClin** [comment]
I think we should add short readmesection for the docker image (I made a test in the mongo DB module)

**Comment by antarus** [comment]
ok I'll look it up

**Comment by antarus** [comment]
@DamnClin 
Isn't it a little redundant to have the documentation and after the docker-compose to launch?

maybe add a specific needle to the docker to start?

**Comment by DamnClin** [comment]
I don't know, most of the time I have many docker to start (and I make a makefile but that's not the subject...), I'dd like to have all the command in the frontpage

**Comment by antarus** [comment]
I agree with you, I regularly make a script to start everything but if you want to put all the useful commands at launch, they would be better under `npm install` in the readme, wouldn't they?

**Comment by DamnClin** [comment]
Totally, good idea

**Comment by antarus** [comment]
which name would be the most appropriate for this needle ? i can do a MR

**Comment by DamnClin** [comment]
sounds good. I think we should also add a shortcut in JHipsterModule where you give the command and it's doing the other part (formatting, replacement, etc)

**Comment by pascalgrimaud** [comment]
> I don't know, most of the time I have many docker to start (and I make a makefile but that's not the subject...), I'dd like to have all the command in the frontpage

For your information, in generator-jhipster: there are some npm commands to start all docker-compose services. See https://github.com/jhipster/jhipster-sample-app/blob/main/package.json#L32-L38

Some users said me they don't like this, as it looks strange to have these kinds of commands inside `package.json`
So here some suggestions, in JHipster Lite:
- a module which add to `package.json` these kind of commands, using docker-compose
- a module which will init a makefile and add the different commands

If you think it's a good idea, we can open a ticket for that

**Pull request merged**